recipes/gst-plugins-bad-1.0.recipe

```diff
@@ -630,7 +629,7 @@ class Recipe(custom.GStreamer):
             self.meson_options['sctp-internal-usrsctp'] = 'disabled'
             self.disable_plugin('soundtouch', 'effects', dep='soundtouch')
             self.disable_plugin('spandsp', 'codecs', dep='spandsp')
-            self.disable_plugin('smoothstreaming', 'codecs', dep='libxml2')
+            self.disable_plugin('smoothstreaming', 'codecs')
```

Comment:
We should have some way to enable or disable plugins without editing cerbero.

Reply:
I agree, but that's out of scope: I didn't invent this recipe, it was already disabling some plugins for web:
```
        if self.config.target_platform == Platform.WEB:
            self.disable_plugin('aes', 'effects')
            self.disable_plugin('assrender', 'codecs', dep='libass')
```
https://github.com/fluendo/cerbero/commit/4ce83e6c10fbfeeb4809eef09528fe66f5b2359b

Here I'm basically just fixing up this commit from the `emscripten` feature (the PR is to).

Also "We should have some way to enable or disable plugins without editing cerbero" sounds more a general request for cerbero not even gst.wasm